src/features/_inline.scss

```diff
@@ -2,3 +2,13 @@
 [style *= "!important"]::after {
   @include contentMessage(warning, "!important inline styles should be avoided.");
 }
+
+* {
+  &:not([role="text"]){
+    > em,span{
+      &:after{
+        @include contentMessage(warning, "Text broken by <span> or <em> elements might need a [role='text'] tag");

```

Comment:
Little nit pick, but could you add a `.` at the end of the sentence?

Reply:
Sure thing!